src/SeleniumLibrary/keywords/element.py

```diff
@@ -409,6 +409,37 @@ def get_element_attribute(
         """
         return self.find_element(locator).get_attribute(attribute)
 
+    @keyword
+    def get_dom_attribute(
+        self, locator: Union[WebElement, str], attribute: str
+    ) -> str:
+        """Returns the value of ``attribute`` from the element ``locator``. `Get DOM Attribute` keyword
+        only returns attributes declared within the element's HTML markup.
```

Comment:
The last sentence here,

> `Get DOM Attribute` keyword only returns attributes declared within the element's HTML markup.

come from the Selenium Python code. Technically this is not actually correct. I found with the tests above, and it would be good to have someone else verify, that if the requested attribute is not there it returns `${None}`. In part that was what I was exploring with the tests. Would it throw an error or return a NaN type response (which Python generally doesn't distinguish from None)? So I suspect we should change this to something else; still explaining what dom attribute is but clarify expected results.

Reply:
Might be good to mention this behavior in the comments.